tendrl/commons/objects/index/__init__.py

```diff
@@ -0,0 +1,58 @@
+import json
+
+import etcd
+
+from tendrl.commons import objects
+from tendrl.commons.utils import etcd_utils
+
+
+class Index(objects.BaseObject):
+
+    def __init__(self, key=None, index_value=None,
+                 *args, **kwargs):
+        super(Index, self).__init__(*args, **kwargs)
+        self._index_name = key.strip("/").rsplit("/", 1)[-1]
+        self.value = "indexes/{0}".format(key.strip("/").rsplit("/", 1)[0])
+        self._is_stale = True
+        self._index_value = index_value
```

Comment:
only one major doubt i have here looks like all attributes are private except value, we are not allowing to store private arrtibutes.